### PR TITLE
node,rpc: remove unused log in ipc listener function

### PR DESCRIPTION
**Pull request by mrFranklin**
there are three changes for the commit:
1. remove "conn.RemoteAddr()" in rpc/ipc.go, because it always `null` for the ipc call.
2. add "IPC" at some places in rpc/ipc.go
3. change "endpoint" -> "url" in node/node.go, just for a consistency with other places.

### CODE REVIEW

File: core/blockchain.go [hunk 1]
```diff
@@ -445,7 +445,11 @@ func (bc *BlockChain) repair(head **types.Block) error {
 			return nil
 		}
 		// Otherwise rewind one block and recheck state availability there
-		(*head) = bc.GetBlock((*head).ParentHash(), (*head).NumberU64()-1)
+		block := bc.GetBlock((*head).ParentHash(), (*head).NumberU64()-1)
+		if block == nil {
+			return fmt.Errorf("failed to repair block, can not get block at height %d", (*head).NumberU64())
+		}
+		(*head) = block
```
Comment (karalabe) [inline core/blockchain.go]: Please remove this part, it's code from a different PR. Please keep PRs self contained.

File: rpc/ipc.go [hunk 2]
```diff
@@ -24,17 +24,17 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/netutil"
 )
 
-// ServeListener accepts connections on l, serving JSON-RPC on them.
+// ServeListener accepts connections on l, serving IPC-RPC on them.
```
Comment (karalabe) [inline rpc/ipc.go]: JSON was correct here.

**Comment by karalabe** [comment]
Ping @mrFranklin

File: core/blockchain.go (hunk 1 above)
Comment (mrFranklin) [reply core/blockchain.go]: sorry for delay, I have fix it. thanks

**Comment by mrFranklin** [comment]
@karalabe  
sorry for delay, I have fix it. thanks.

**Comment by mrFranklin** [comment]
@karalabe
could you review this, thanks. :)

**Pull request merged**